plugins/catalog-backend/config.d.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  /** Configuration options for the catalog plugin */
+
+  catalog?: {
+    rules?: Array<{
```

Comment:
https://github.com/backstage/backstage/blob/master/plugins/catalog-backend/src/ingestion/CatalogRules.ts#L40

Reply:
Thank you @freben! will do the needful!